docs/helpful-resources/ogm-metadata-basics/fields.md

```diff
@@ -21,9 +21,9 @@ Important field elements and special fields worth noting
 
 ---
 
-See the [current schema](../../current-schema/ogm-aardvark) for a full description of fields, expected values, and obligations.
+See the [current schema](../../../current-schema/ogm-aardvark) for a full description of fields, expected values, and obligations.
 
-## Field Elements
+## Field elements
```

Comment:
That makes sense! I like your suggestion, or perhaps "How fields are named"? 

Could you also update the subtitle for the page then? I was having trouble with a clever way to explain the purpose of this page.

Reply:
Maybe we could leave the subtitle blank for now? 

I am now realizing that we are are slipping into a lot of nesting and TOCs that might be unintuitive. When I read about Best Practices for menus, it is not recommended to have multiple nested pages.

Reply:
I agree that having grand-child pages isn't ideal. This was my original reason for suggesting we condense all the Aardvark Schema individual pages into a single page, like the GBL 1.0 page, so that there weren't grand-children under Current Schema.

Can I propose we merge this commit for now, and get feedback from others (perhaps from the coding team) on how to restructure the pages again? This takes a lot of work each time restructuring happens so it'd be nice to come up with a finalized plan before making more changes.

Reply:
Sure! All the info is here and I anticipate that we will add more over time, which may necessitate adjustments anyways.